coda-automation/terraform/testnets/funky-walnut/main.tf

```diff
@@ -0,0 +1,54 @@
+terraform {
```

Comment:
Worth getting rid of (and not migrating) some of the old/stale testnet DIRs? Not sure I see the purpose of keeping them around other than for examples (also doable via something like git history, archive directory etc)?

Reply:
Yep. Cool with you if that's a follow-up pr? They'll be in the history then at least (and makes this pr smaller which I like)